plugins/DuskVerb/tools/tuner/vintage_minisweep2.py

```
@@ -0,0 +1,93 @@
+        "--param", f"Mid Multiply={mid}",
+        "--param", f"Damping={treble}",
+        "--param", f"Bass Multiply={bass}",
+        "--param", f"Low Crossover={lo_xover}",
+        "--param", f"Hi Cut={hi_cut}",
+        "--param", f"Gain Trim={trim}",
+        "--prerun-seconds", "5",
+        "--sustained-pink-seconds", "4",
+        "--output-dir", str(out_dir),
+    ]
+    try:
+        subprocess.run(cmd, capture_output=True, text=True, timeout=120)
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)